Sources/ArcGISToolkit/Components/Popups/Attachments/AttachmentsPopupElementModel.swift

```diff
@@ -71,7 +71,7 @@ import QuickLook
             try await self.attachment.load()
             
             let request = QLThumbnailGenerator.Request(
-                fileAt: attachment.fileURL,
+                fileAt: attachment.fileURL!,
```

Comment:
My apologies, I didn't notice the method was called in an unstructured task block hence the error is discarded. I thought it would throw an error when the load method fails. 🤦